app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -1162,7 +1162,9 @@ DeviceDelegateOpenXR::EndFrame(const FrameEndMode aEndMode) {
       frameEndInfo.environmentBlendMode = blendMode;
       frameEndInfo.layerCount = (uint32_t) layers.size();
       frameEndInfo.layers = layers.data();
-      CHECK_XRCMD(xrEndFrame(session, &frameEndInfo));
+      if (XR_FAILED(MessageXrResult(xrEndFrame(session, &frameEndInfo)))) {
+          VRB_WARN("OpenXR xrEndFrame has failed !!! This means some serious issue in the integration with the OpenXR runtime");
```

Comment:
The MessageXr already checks XR_FAILED and prints a message you don't need them

Reply:
We have a lot of these logs and I wanted to highlight this a bit more; but if you don't see the need, I'll just print the result.